devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -2083,6 +2377,30 @@ bool WholeBodyDynamicsDevice::resetOffset(const std::string& calib_code)
     return true;
 }
 
+bool WholeBodyDynamicsDevice::usePreEstimatedOffset()
+{
+    yarp::os::LockGuard guard(this->deviceMutex);
+
+    yWarning() << "wholeBodyDynamics : using offset estimated offline.";
+
+    for(size_t ft = 0; ft < this->getNrOfFTSensors(); ft++)
+    {
+        double norm = iDynTree::toEigen(ftProcessors[ft].estimatedOffset()).norm();
+        if (norm!= 0.0){ // if norm is exactly 0.0 it means there was no offset  in the configuration file so it should be skipped
```

Comment:
this seems dangerous ;D This needs to be documented somewhere visible to the user.

Reply:
Added in the description of the function in the .h file